### PR TITLE
feat(FX-3110): list of saved search alerts

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Feature**

This PR resolves [FX-3110]

### Description
* As an admin with the saved search M2 feature flag enabled,
* I can navigate to the profile screen
* And see a link "saved alerts"
* And tapping that link takes me to a screen where all of my saved alerts are listed
* In the current version, we decided not to display the filter labels. You can find more context [here](https://artsy.slack.com/archives/C9SATFLUU/p1628072996113000?thread_ts=1627902236.061100&cid=C9SATFLUU)

### Demo
https://user-images.githubusercontent.com/3513494/128221242-64d96efd-e861-4f68-9de1-a9ad64b22363.mp4

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- list of saved search alerts - dzmitry tratsiak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[FX-3110]: https://artsyproduct.atlassian.net/browse/FX-3110

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (list of saved search alerts - dzmitry tratsiak)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 98945bcebed24ccc50326b2add35355ada81d0d3
</p>

File: src/lib/data/constants.ts [hunk 1]
```diff
@@ -4,3 +4,4 @@ export const FAIR2_ARTWORKS_PAGE_SIZE = 30
 export const FAIR2_EXHIBITORS_PAGE_SIZE = 30
 export const ARTIST_SERIES_PAGE_SIZE = 30
 export const SHOW2_ARTWORKS_PAGE_SIZE = 30
```
Comment (pvinis) [inline src/lib/data/constants.ts:6]: wow, i actually have never seen this file before. weird.

File: src/lib/Scenes/SavedSearchAlertsList/Components/SavedSearchesList.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    <FlatList
+      data={items}
+      keyExtractor={(item) => item.internalID}
+      contentContainerStyle={{ paddingVertical: space(1) }}
+      renderItem={({ item }) => {
+        return (
+          <SavedSearchListItem
+            title={item.userAlertSettings.name!}
+            onPress={() => {
+              console.log("pressed")
```
Comment (pvinis) [inline src/lib/Scenes/SavedSearchAlertsList/Components/SavedSearchesList.tsx:49]: is this ok or forgotten to be replaced?

**Review by pvinis — APPROVED** [review]

File: src/lib/Scenes/SavedSearchAlertsList/Components/SavedSearchesList.tsx (hunk 2 above)
Comment (dimatretyak) [reply src/lib/Scenes/SavedSearchAlertsList/Components/SavedSearchesList.tsx:49]: Ok, this will be replaced in the near future

**Pull request merged**